src/Framework/UnitTests/project.json

```diff
@@ -15,14 +15,14 @@
     },
     "netstandard1.3": {
       "dependencies": {
-        "System.Net.NetworkInformation": "4.1.0-rc3-24109-00",
-        "Microsoft.NETCore": "5.0.1-rc3-24109-00",
-        "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24109-00",
-        "System.Collections.NonGeneric": "4.0.1-rc3-24109-00",
-        "System.Console": "4.0.0-rc3-24109-00",
-        "Microsoft.NETCore.Runtime": "1.0.2-rc3-24109-00",
-        "Microsoft.NETCore.TestHost": "1.0.0-rc3-24109-00",
-        "Microsoft.Win32.Primitives": "4.0.1-rc3-24109-00",
+        "System.Net.NetworkInformation": "4.1.0",
+        "Microsoft.NETCore": "5.0.1",
+        "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+        "System.Collections.NonGeneric": "4.0.1",
+        "System.Console": "4.0.0",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+        "Microsoft.NETCore.TestHost": "1.0.0",
+        "Microsoft.Win32.Primitives": "4.0.1",
         "xunit.console.netcore": "1.0.2-prerelease-00410-02"
```

Comment:
Unfortunately, there's no RTM version of those. But you're probably right that it's worth moving them to be more recent. I'll give it a shot.